src/main/java/frc/robot/subsystems/swerve/mk4iswerve/mk4imodule/MK4IModuleActions.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.swerve.mk4iswerve.mk4imodule;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.utils.devicewrappers.GBTalonFXPro;
+
+public class MK4IModuleActions {
+
+    private final GBTalonFXPro steerMotor, driveMotor;
+
+    private final VelocityVoltage driveVelocityRequest = new VelocityVoltage(0);
+    private final VoltageOut driveVoltageRequest = new VoltageOut(0).withEnableFOC(MK4IModuleConstants.ENABLE_FOC_DRIVE);
+    private final PositionVoltage steerPositionRequest = new PositionVoltage(0).withEnableFOC(MK4IModuleConstants.ENABLE_FOC_STEER);
+
+
+    public MK4IModuleActions(GBTalonFXPro driveMotor, GBTalonFXPro steerMotor){
+        this.driveMotor = driveMotor;
+        this.steerMotor = steerMotor;
+    }
+
+
+    public void setTargetOpenLoopVelocity(double voltage) {
+        driveMotor.setControl(driveVoltageRequest.withOutput(voltage));
+    }
+
+
+    public void setTargetClosedLoopVelocity(double targetVelocityMetersPerSecond) {
+        driveMotor.setControl(driveVelocityRequest.withVelocity(targetVelocityMetersPerSecond));
+    }
+
+
+    public void setTargetAngle(Rotation2d angle) {
+        steerMotor.setControl(steerPositionRequest.withPosition(angle.getRotations()));
+    }
+
+
+    public void resetSteerAngle(Rotation2d angle) {
+        steerMotor.setPosition(angle.getRotations());
+    }
+
+
+    public void stop() {
+        steerMotor.stopMotor();
+        driveMotor.stopMotor();
+    }
+
+
+    public void setBrake(boolean brake) {
+        final NeutralModeValue neutralModeValue = brake ? NeutralModeValue.Brake : NeutralModeValue.Coast;
```

Comment:
![image](https://github.com/GreenBlitz/Off-Season-2024/assets/139997639/29d510a1-6655-43cf-b924-610e173a8fbe)
```suggestion
        NeutralModeValue neutralModeValue = brake ? NeutralModeValue.Brake : NeutralModeValue.Coast;
```

Reply:
i dont agree with that

Reply:
I don't agree with that